src/apdu_pubkey.c

```diff
@@ -73,9 +73,8 @@ int handle_get_public_key(buffer_t *cdata,
     TZ_ASSERT_NOT_NULL(cdata);
 
     global.path_with_curve.derivation_type = derivation_type;
-
     if ((cdata->size == 0u) && authorize) {
-        TZ_ASSERT(copy_bip32_path_with_curve(&global.path_with_curve, &N_data.baking_key),
+        TZ_ASSERT(copy_bip32_path_with_curve(&global.path_with_curve, &(g_hwm.baking_key)),
```

Comment:
`g_hwm` is not an alias for `N_data`.
Why using `hwm_data` (formerly new_data) rather than `N_data`?

Reply:
g_hwm is alias for object stored in RAM where as N_data is stored in NVRAM. Using g_hwm is faster here. plus when hwm is disabled N_data becomes outdated so can not be used.